test/test_phone_number_ru.rb

```diff
@@ -0,0 +1,76 @@
+# encoding: utf-8
+
+require_relative 'helper'
+
+class TestPhoneNumberRU < Test::Unit::TestCase
+  include DeterministicHelper
+
+  assert_methods_are_deterministic(
+    FFaker::PhoneNumberRU,
+    :international_phone_number, :international_mobile_phone_number,
+    :international_home_work_phone_number, :international_toll_free_number,
+    :phone_number, :mobile_phone_number, :home_work_phone_number,
+    :toll_free_number, :international_country_code, :country_code,
+    :home_work_phone_prefix, :mobile_phone_prefix
+  )
+
+  def setup
+    @tester = FFaker::PhoneNumberRU
+  end
+
+  def test_international_phone_number
+    assert_match(/\+7 \d{3} \d{3}-\d{2}-\d{2}/,
```

Comment:
this regexp pass: 
- +7 123 45-67-890
- ++7 123 45-67-89

```suggestion
    assert_match(/\A\+7 \d{3} \d{3}-\d{2}-\d{2}\z/,
```

Reply:
Done.